trade-module/src/main/java/io/edpn/backend/trade/application/domain/Station.java

```diff
@@ -20,4 +20,9 @@ public record Station(
         LocalDateTime marketUpdatedAt,
         List<MarketDatum> marketData
 ) {
+    public Station {
+        if (maxLandingPadSize == null) {
+            maxLandingPadSize = LandingPadSize.UNKNOWN;
+        }
```

Comment:
Also needs:
        if (marketData == null) {
            marketData = new ArrayList<>();
        }
        
   To retain same functionality as previous